CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 
 # Unreleased
 
+- Export the functions: `createObjectAddress` `createResourceAddress` `createTokenAddress`
```

Comment:
we actually export everything under the `account` folder, right?

Reply:
Yes, the account folder only contains these functions, and they will be used frequently.

Reply:
I added the account folder to the changelog, so it may be more accurate.